src/main/java/com/checkmarx/jenkins/CheckmarxScanBuilder.java

```diff
@@ -356,6 +356,31 @@ public void close() throws IOException {
             ArtifactArchiver artifactArchiverJson = new ArtifactArchiver(workspace.toURI().relativize(jsonReportFilePath.toURI()).toString());
             artifactArchiverJson.perform(run, workspace, envVars, launcher, listener);
 
+            if (scanConfig.getAdditionalOptions().contains("--report-format")) {
```

Comment:
maybe set it in another private method with a name that will describe what you ar doing here